test/integration/test-example-validation.js

```diff
@@ -30,7 +30,7 @@ if (!window.validatorLoad) {
   })();
 }
 
-describe.configure().retryOnSaucelabs().run('example', function() {
+describe.skip('example', function() {
```

Comment:
Just verified that `gulp test` (which runs on Travis with latest Chrome as part of the pre-submit and unit test job) does indeed run integration tests. (it runs `test/**/*`)